Client/UserResearch/NewTabUserResearch.swift

```diff
@@ -47,77 +47,10 @@ class NewTabUserResearch {
-        guard LeanPlumClient.shared.getSettings() != nil else {
-            // default state is false
-            self.newTabState = false
-            return
-        }
-        // Condition: A/B test variables from leanplum server
-        LeanPlumClient.shared.finishedStartingLeanplum = {
-            LeanPlumClient.shared.finishedStartingLeanplum = nil
-            guard self.fetchedExperimentVariables == false else {
-                return
-            }
-            self.fetchedExperimentVariables = true
-            //Only update add new tab (+ button) when it doesn't match leanplum value 
-            let lpValue = LPVariables.newTabButtonABTest?.boolValue() ?? false
-            if self.newTabState != lpValue {
-                self.newTabState = lpValue
-                self.updateTelemetry()
-            }
-        }
-        // Condition: Leanplum server too slow; Set default New tab state
-        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
-            guard self.fetchedExperimentVariables == false && self.newTabState == nil else {
-                return
-            }
-            // Condition: Leanplum server too slow; Set default New tab state
-            self.newTabState = false
-            // Condition: LP has already started but we missed onStartLPVariable callback
-            if case .started(startedState: _) = LeanPlumClient.shared.lpState , let boolValue = LPVariables.newTabButtonABTest?.boolValue() {
-                self.newTabState = boolValue
-                self.updateTelemetry()
-            }
-            self.fetchedExperimentVariables = true
-        }
-    }
-    
     func updateTelemetry() {
         guard !hasEnrolled else { return }
-        // Printing variant is good to know all details of A/B test fields
-        print("lp variant \(String(describing: Leanplum.variants()))")
-        var lpData: Dictionary<String, Any>?
-        guard let variants = Leanplum.variants() as? [Dictionary<String, Any>] else {
-            return
-        }
-        variants.forEach {
-            if $0["abTestName"] as? String == abTestName {
-                lpData = $0
-            }
-        }
-        guard lpData != nil else {
-            return
-        }
-        var abTestId = ""
-        if let value = lpData?["abTestId"] as? Int64 {
-                abTestId = "\(value)"
-        }
-        let abTestVariant = lpData?["name"] as? String ?? ""
-        let attributesExtras = [LPAttributeKey.experimentId: abTestId, LPAttributeKey.experimentName: abTestName, LPAttributeKey.experimentVariant: abTestVariant]
-        // Leanplum telemetry
-        LeanPlumClient.shared.set(attributes: attributesExtras)
-        // Legacy telemetry
-        TelemetryWrapper.recordEvent(category: .enrollment, method: .add, object: .experimentEnrollment, extras: attributesExtras)
+        
+        TelemetryWrapper.recordEvent(category: .enrollment, method: .add, object: .experimentEnrollment, extras: nil)
```

Comment:
```suggestion
```
@nishant2718 I'd suggest removing this as well. Since we no longer add extras and without Leanplum AB test it'll only send `.enrollment` events.